src/app/features/main-page/components/footer/footer.component.scss

```
@@ -0,0 +1,95 @@
+@use '../../../../../style/abstract/constant' as *;
+@use '../../../../../style/abstract/placeholders' as *;
+
+footer {
+  box-shadow: 0 0 10px 5px $color-input-search;
+}
+.footer {
+  display: flex;
+  justify-content: space-between;
+  column-gap: 1.5rem;
+  padding: 1rem;
+  font-size: 1.6rem;
+  font-weight: 600;
+  color: $color-additional;
+
+  &__contacts {
+    display: flex;
+    flex-direction: column;
+    row-gap: 1.5rem;
+    font-size: 1.2rem;
+    align-items: center;
+
+    .header {
+      font-size: 1.6rem;
+      text-decoration: underline;
+    }
```

Please, remove the underline style, because it is not a clickable element.

```suggestion
    .header {
      font-size: 1.6rem;
   
    }
```